sdk/serialization/azure-xml/src/main/java/com/azure/xml/XmlReader.java

```diff
@@ -434,12 +455,35 @@ public long getLongElement() throws XMLStreamException {
      * If the current element doesn't have a value null will be returned, otherwise the element
      * {@link #getStringElement() text value} is passed to the converter.
      *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * <!-- src_embed com.azure.xml.XmlReader.getNullableElement#ReadValueCallback -->
+     * <pre>
+     * try &#40;XmlReader reader = XmlReader.fromString&#40;&quot;&lt;root&gt;&lt;element&gt;1234&lt;&#47;element&gt;&lt;emptyElement&#47;&gt;&lt;&#47;root&gt;&quot;&#41;&#41; &#123;
+     *     reader.nextElement&#40;&#41;; &#47;&#47; Progress to &lt;root&gt;
+     *     reader.nextElement&#40;&#41;; &#47;&#47; Progress to &lt;element&gt;
+     *
+     *     &#47;&#47; Get the value of the element &quot;element&quot; as an Integer in a way that allows for the element to be missing
+     *     &#47;&#47; or have a null value.
+     *     assertEquals&#40;1234, &#40;int&#41; reader.getNullableElement&#40;Integer::parseInt&#41;&#41;; &#47;&#47; 1234
```

Comment:
Do we want to have `assertEquals` or any junit APIs in javadocs?

Reply:
I'm not against it as it does clearly show code behaviors, and in a better way than we do now with print lines